classes/pvroot-image.bbclass

```diff
@@ -25,7 +25,7 @@ python __anonymous () {
     pn = d.getVar("PN")
 
     for img in d.getVar("PVROOT_IMAGE_BSP").split():
-        d.appendVarFlag('do_rootfs', 'depends', ' '+img+':do_image_complete')
+        d.appendVarFlag('do_rootfs', 'depends', ' '+img+':do_image_complete '+img+':do_image_pvbspit')
```

Comment:
this seems to not match the PR title ... does this have any effect/ do_image_complete should depend on this already if the IMAGE_CLASSES have the bspimage-pvrexport class.

Reply:
Its true that do_image_complete depends on do_image_pvbspit, and yet I had an issue here with the file not existing. Hopefully its better now that the file is actually being 'deployed' correctly during do_image_complete. 
Having said that, adding do_image_pvbspit as a dependency here will flag an error of the BSP image does not inherit the "bspimage-pvrexport" class, which is just free insurance.
This ties in with another point I wanted to make: I prefer not to include `IMAGE_CLASSES += " bspimage-pvrexport "` being in a top-level conf file. This adds a few minutes build time to any image not related to pantavisor. Each implementation can still pick their own BSP provider, and get a warning if none exists: 
``` Bitbake 
# local.conf:
PVROOT_IMAGE_BSP ?= "fsl-image-network-full-cmdline"
IMAGE_CLASSES:append:pn-fsl-image-network-full-cmdline = " bspimage-pvrexport "
```

Reply:
let me test a few combinations ... i would be surpirsed to see that this works without having bspimage-pvrexport in IMAGE_CLASSES .... as another idea to speed up, we could however somehow mask ourselves if we are not in the PVROOT_IMAGE_BSP receipt.... also we could try to mask all IMAGE_CLASSES that are != bspimage-pvrexport and != image-pvrexport in case we are in thePVROOT_IMAGE_BSP receipt.